extensions/amp-base-carousel/0.1/child-layout-manager.js

```diff
@@ -165,6 +168,9 @@ export class ChildLayoutManager {
     /** @private {?IntersectionObserver} */
     this.inViewportObserver_ = null;
 
+    /** @private {!JsonObject<string, Element>} */
+    this.underlyingAmpElementParent_ = {};
```

Comment:
This is a mapping between SLIDE_ID and the corresponding slide element?

Reply:
Yes, I'll rename it to be more descriptive.